openapi/components/schemas/Common/CommonTransaction.yaml

```
@@ -138,67 +152,75 @@ properties:
     allOf:
       - $ref: ../Contact/ContactObject.yaml
   has3ds:
+    description: Specifies whether the transaction uses 3D Secure.
     type: boolean
     readOnly: true
   3ds:
+    description: 3D Secure object.
     allOf:
       - $ref: ../Transactions/ThreeDSecureResult.yaml
   redirectUrl:
-    description: The URL to redirect the end-user when an offsite transaction is completed. Defaults to the website's configured URL.
+    description: |-
+      URL where the end-user is redirected when an offsite transaction
```

I think this is a bit clearer
```suggestion
      URL where the end-user is redirected to when an offsite transaction
```

---

Looks good